src/modules/aws_cloudwatch/entity/metric_alarm.ts

```diff
@@ -316,8 +317,9 @@ export class MetricAlarm {
    * @public
    * The value against which the specified statistic is compared.
    */
-  @Column({
+  @Column('numeric', {
```

Comment:
So according to the documentation this field is [basically a `double](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-cloudwatch/interfaces/putmetricalarmcommandinput.html#threshold)

Instead of `numeric` and then converted it internally into a double, why not use the [`double precision`](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-FLOAT) type in Postgres directly and avoid the conversion noise?